### PR TITLE
small brain === nox

### PR DESCRIPTION
**Pull request by Noxemia**
NOW RUNS

### CODE REVIEW

File: application/app/src/main/java/com/yahyeet/boardbook/model/Boardbook.java [hunk 1]
```diff
@@ -33,15 +33,15 @@ public Boardbook(IAuthService authService,
                      IChatMessageRepository chatMessageRepository) {
 
         /// Use this for dependency injection for mock databases
-        if(TEST_FLAG = false) {
+
```
Comment (cajoho99) [inline application/app/src/main/java/com/yahyeet/boardbook/model/Boardbook.java]: Why newline?

File: application/app/src/main/java/com/yahyeet/boardbook/model/Boardbook.java [hunk 2]
```diff
@@ -33,15 +33,15 @@ public Boardbook(IAuthService authService,
...
             this.userHandler = userHandler;
             gameHandler = new GameHandler(gameRepository);
             matchHandler = new MatchHandler(matchRepository);
             chatGroupHandler = new ChatGroupHandler(chatGroupRepository);
             chatMessageHandler = new ChatMessageHandler(chatMessageRepository);
 
             authHandler = new AuthHandler(authService, userHandler);
-        }
+
```
Comment (cajoho99) [inline application/app/src/main/java/com/yahyeet/boardbook/model/Boardbook.java]: Why newline?

File: application/app/src/main/java/com/yahyeet/boardbook/model/Boardbook.java (hunk 1 above)
Comment (rosengrenen) [reply application/app/src/main/java/com/yahyeet/boardbook/model/Boardbook.java]: new line = bad code

**Comment by Noxemia** [comment]
Fuck ya'll

**Pull request merged**